src/test/java/com/scalefocus/libraryproject/BasicTests.java

```diff
@@ -0,0 +1,48 @@
+package com.scalefocus.libraryproject;
+
+import com.scalefocus.libraryproject.entities.UserEntity;
+import com.scalefocus.libraryproject.enums.Role;
+import com.scalefocus.libraryproject.models.UserModel;
+import com.scalefocus.libraryproject.repositories.UserRepository;
+import com.scalefocus.libraryproject.services.UserService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+@SpringBootTest
+public class BasicTests {
+
+    @MockBean
+    private UserRepository userRepository;
+    @Autowired
+    private UserService userService;
+    @Test
+    public void updateUserTest() {
```

Comment:
Please, update the test's name to denote it's purpose more precisely. E.g.
if it's a possitive scenario test you can rename it to
updateUserTest_success()
or if it's a negative test you can specify how it fails:
updateUserTest_missingDatabaseRecord_failure()